torchvision/csrc/ROIAlign.h

```diff
@@ -32,8 +34,29 @@ at::Tensor roi_align(
       pooled_height,
       pooled_width,
       sampling_ratio,
+      aligned).to(input.scalar_type());
```

Comment:
I would have expected this casting to happen in the `ROIAlign_autocast` implementation?

Reply:
Agreed that makes more sense.